tutorials/napps/development_environment_setup.rst

```diff
@@ -38,16 +38,67 @@ Installing required dependencies
 ********************************
 
 In order to start using and coding with Kytos, you need a few required
-dependencies. One of them is Python 3.6 or greater.
+dependencies. One of them is Python 3.9.x
 
+
+Installing Python 3.9.x
+=====================
+
+Install the dependencies necessary to build Python:
```

Comment:
Maybe we could add a disclaimer that users should install Python 3.9 from scratch if their distro don't provide a package for it? Something lke: `If your OS distribution doesn't provide Python 3.9 and development packages, you can install Python 3.9 from source by following these instructions`

What you think?

Reply:
yes, we can add, but then we have to add the instructions for both scenarios. or further steps to check if their OS distribution provides or not. that might be confusing. we can just simply get him to update yo python3 using these steps. that's what i think

Reply:
Right. Yeah, so, let's keep as you have here, and installing from source would be what they'll use if they their OS don't provide the 3.9 packages, this also simplifies the maintenance tutorial on our end. So, with that being said, we should probably remove this part here -> `apt install git libpython3.9-dev python3.9 python3.9-venv` and only leave `git` since the 3.9 packages below aren't provided by default on Ubuntu and probably depending on a ppa can become a maintenance concern in the future. What do you think? Thanks @usmanazFIU